packages/perseus/src/util.ts

```diff
@@ -919,6 +919,23 @@ const unescapeMathMode: (label: string) => string = (label) =>
 
 const random: RNG = seededRNG(new Date().getTime() & 0xffffffff);
 
+// TODO(benchristel): in the future, we may want to make deepClone work for
+// Record<string, Cloneable> as well. Currently, it only does arrays.
+type Cloneable =
+    | null
+    | undefined
+    | boolean
+    | string
+    | number
+    | Cloneable[]
+    | readonly Cloneable[];
+function deepClone<T extends Cloneable>(obj: T): T {
+    if (Array.isArray(obj)) {
+        return obj.map(deepClone) as T;
+    }
+    return obj;
```

Comment:
This feels a bit misleading. We aren't doing any cloning here, just returning the original value (which I guess is correct given the types that Cloneable currently covers).  

We do already have a `clone` (which is actually a deep clone) in https://github.com/Khan/perseus/blob/53fd3768652b702df4a79b97a0ab4766b11748fe/testing/object-utils.ts#L3 

Could we rationalize these two implementations?